installing/installing_gcp/installing-gcp-vpc.adoc

```diff
@@ -0,0 +1,46 @@
+[id="installing-gcp-vpc"]
+= Installing a cluster on GCP to an existing VPC
+include::modules/common-attributes.adoc[]
+:context: installing-gcp-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster into an existing Virtual Private Cloud (VPC) on Google Cloud Platform (GCP). The installation program provisions the rest of the required infrastructure, which you can further customize. To customize the installation, you modify
```

Comment:
I haven't noticed where we include "version" before {product-version}. I don't think it's necessary, unless I'm missing a guideline, which is certainly possible!

Reply:
It was a convention at IBM, but I don't see it mentioned in the ISG.